zipline/algorithm.py

```diff
@@ -193,10 +192,12 @@ class TradingAlgorithm(object):
         default: 'zipline'
     """
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, env, *args, **kwargs):
         """Initialize sids and other state variables.
 
         :Arguments:
+        :Required:
```

Comment:
the __init__ docstring should be removed the args should be documents in the class docstring